crates/ruff_python_formatter/src/options.rs

```diff
@@ -49,6 +49,12 @@ pub struct PyFormatOptions {
     /// enabled by default (opt-out) in the future.
     docstring_code: DocstringCode,
 
+    /// The preferred line width at which the formatter should wrap lines in
+    /// docstring code examples. This only has an impact when `docstring_code`
+    /// is enabled.
+    #[cfg_attr(feature = "serde", serde(default = "default_line_width"))]
+    docstring_code_line_width: LineWidth,
```

Comment:
I'd expect this to default to the formatter's line-length, but I _think_ that happens at a level above, when we create the `Settings`.

Reply:
The default initialization happens inside `into_settings`

https://github.com/astral-sh/ruff/blob/241429828905ef1287a53423f0c6f12d52fba313/crates/ruff_workspace/src/configuration.rs#L173-L192